conf/distro/include/ostro-xt-security-flags.inc

```diff
@@ -0,0 +1,3 @@
+SECURITY_CFLAGS_pn-crosswalk = "${SECURITY_NO_PIE_CFLAGS} ${SECURITY_PIC_CFLAGS}"
+SECURITY_LDFLAGS_remove_pn-crosswalk = "-fstack-protector-strong"
+SECURITY_LDFLAGS_append_pn-crosswalk = " -fstack-protector-all"
```

Comment:
I refer to https://github.com/ostroproject/ostro-os-xt/blob/master/meta-ostro-xt/conf/distro/include/ostro-xt-security-flags.inc#L25. Do you have any suggestions?